core/src/main/java/utilcalc/core/reportGen/Format.java

```diff
@@ -0,0 +1,7 @@
+package utilcalc.core.reportGen;
+
+public enum Format {
```

Comment:
Kdyz uz je to v package `reportGen`, tak bych cekal, ze se to v nem bude i pouzivat. Bud bych tohle presunul do web, nebo naopak bych do ReportGen presunul `ReportGeneratorService::generate(ReportInputs, Format)`. Vlastne hodne podobny switch mame i v CLI, takze by to chtelo zkonsolidovat.